roxie/ccd/ccdqueue.cpp

```diff
@@ -746,6 +778,11 @@ class RoxieQueue : public CInterface
             queueLength++;
             if (queueLength>maxQueueLength)
                 maxQueueLength = queueLength;
+            if (idle==0 && started < numWorkers)
```

Comment:
I suspect avoidance of race conditions was the motivation for pre-starting the threads in the first place.